setup.cfg

```
@@ -25,7 +25,7 @@ package_dir =
     = src
 python_requires = >=3.7
 install_requires =
-    pyhf[minuit]==0.5.4  # paramset.suggested_fixed
+    pyhf[minuit]~=0.6
```

```suggestion
    pyhf[minuit]>=0.6.3
```

@kratsg we changed to `include_auxdata` in [`pyhf` `v0.6.3`](https://github.com/scikit-hep/pyhf/releases/tag/v0.6.3), so at the very least this should be `~=0.6,>=0.6.3`, but I think it would be more helpful to just do `>=0.6.3` here.